options/options.c

```diff
@@ -715,6 +715,10 @@ const m_option_t mp_opts[] = {
                       0, INT_MAX, ({"auto", -1})),
 #endif
 
+#if HAVE_VAAPI_HWACCEL
+    OPT_STRING("vaapi-device", vaapi_device, M_OPT_FILE),
```

Comment:
I'm not too sure about this. Most likely the option would have to be replaced later anyway. Since this is a relatively obscure option, merging them would probably be ok for now.